src/app/actions/upload.ts

```diff
@@ -29,9 +30,19 @@ export async function uploadCsv(file: FormDataEntryValue | null, userId: number)
       throw error;
     }
 
-    const searchResultPayload = keywords.map((keyword) => ({ keyword: keyword, userId: +userId }));
+    const payload = keywords.reduce(
+      (accumulator: Prisma.SearchResultCreateManyInput[], keyword) => {
+        accumulator.push(
+          { keyword, userId: +userId, searchEngine: SearchEngine.bing },
+          { keyword, userId: +userId, searchEngine: SearchEngine.bing }
```

Comment:
One of them should be Google, right? This would enqueue later the scarping for Bing twice. 

```suggestion
          { keyword, userId: +userId, searchEngine: SearchEngine.google }
```